### PR TITLE
#12 - Add CDK Bootstrap s3 bucket 

### PR DESCRIPTION
**Pull request by jbnaik**
Changes should create new stack called `CDKToolkit` with encrypted bucket in each AWS account.

### CODE REVIEW

File: cdk-test/core/bin/main.ts [hunk 1]
```diff
@@ -2,12 +2,21 @@
...
 import { App } from '@aws-cdk/core';
 import { CosmosStack, GalaxyStack, CiCdSolarSystemStack, EcsSolarSystemStack } from '@cdk-cosmos/core';
+import { BootstrapStack } from '@cosmos-building-blocks/common';
 
 const app = new App();
 
 const mgtEnvConfig = { account: '1111', region: 'ap-southeast-2' };
 const devEnvConfig = { account: '2222', region: 'ap-southeast-2' };
 
+const bootstrapMgt = new BootstrapStack(app, 'CDKToolkitMgt', {
```
Comment (timpur) [inline cdk-test/core/bin/main.ts]: Maybe `CDKToolkit` as the class name ?

File: packages/@cosmos-building-blocks/common/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK TypeScript Construct Library project!
```
Comment (timpur) [inline packages/@cosmos-building-blocks/common/README.md]: delete this for now, will re-add int the doc PR

File: packages/@cosmos-building-blocks/common/test/lib-template.test.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { expect as expectCDK, haveResource, SynthUtils } from '@aws-cdk/assert';
+import * as cdk from '@aws-cdk/core';
+import LibTemplate = require('../lib/index');
+
+test('SQS Queue Created', () => {
```
Comment (timpur) [inline packages/@cosmos-building-blocks/common/test/lib-template.test.ts]: does this work ?

Comment (jbnaik) [reply packages/@cosmos-building-blocks/common/test/lib-template.test.ts]: removed test

File: packages/@cosmos-building-blocks/common/README.md (hunk 2 above)
Comment (jbnaik) [reply packages/@cosmos-building-blocks/common/README.md]: done

File: cdk-test/core/bin/main.ts (hunk 1 above)
Comment (jbnaik) [reply cdk-test/core/bin/main.ts]: Done

**Pull request merged**